### DISCUSSION TITLE
Kindly please add Flatpak support - this would enable millions of kids easy access to LLM's

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#1617] [General]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing **Text generation web UI** on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

Please support this initiative as it would turn the world's number 1 School/Education focused Linux distro "Endless OS" into a truely endless OS.

https://www.endlessos.org/os

**Comment by BetaDoggo** [comment]
I doubt many children/schools have the hardware required to run language models locally. Most schools would probably stick to the proprietary "safe" services anyway to avoid liability for any "unsafe" generations.
